### DISCUSSION TITLE
Pushed changes not applied

### DISCUSSION THREAD

**Discussion by dougpiercey** [opengisch/QField#4165] [Q&A]
Relatively new to QField/QCloud. We have been testing over the past couple of months and see good things with this as a field data collection solution.

Lately, I have been having an issue with pushed attribute changes not being applied. Initially, the Status for these changes was "Error", but after trying to apply all the changes they are now listed as "Not_Applied". All of the changes have the same info - "No layer with id "CB_Treed_Plots_c528d9de_b380_4fc7_9635_a48a314321fd"

![image](https://user-images.githubusercontent.com/105067319/231782247-22d258b6-eaf9-43de-b3d0-9f9ae7224ae4.png)

Here is some detail from the Job info...

![image](https://user-images.githubusercontent.com/105067319/231783103-fa611815-2195-4872-a7e0-ddc438dfb126.png)

The layer in the geopackage is CBTreedPlots (no underscores) and this hasn't changed.

Any ideas as to what could be causing this?

Thanks

**Comment by idantene** [comment]
We've encountered a similar issue.
My understanding is that at some point, the layer in question was reloaded in a client, it received a different `layer_id`, and was then pushed. Since nothing in the layer actually changed, QField did not sync this new ID, and any updates on a different device use a different `layer_id`, which no longer exists, and so changes cannot be applied.

Have you found a way to resolve this?
